src/main/kotlin/com/example/noswimmingserver/domain/feedback/domain/Feedback.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.noswimmingserver.domain.feedback.domain
+
+import com.example.noswimmingserver.domain.reading_journal.domain.ReadingJournal
+import com.example.noswimmingserver.domain.teacher.domain.Teacher
+import java.time.LocalDateTime
+import javax.persistence.*
```

Comment:
🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)